libs/auth/scopes/src/lib/auth.scope.ts

```diff
@@ -6,7 +6,8 @@ export enum AuthScope {
   adminPersonalRepresentative = '@island.is/auth/personal-representative-admin',
   publicPersonalRepresentative = '@island.is/auth/personal-representative-public',
   consents = '@island.is/auth/consents',
-  delegationIndexWrite = '@island.is/auth/delegation/index:write',
+  delegationIndexWrite = '@island.is/auth/delegations/index:write',
+  delegationIndex = '@island.is/auth/delegations/index',
```

Comment:
We should create migration scripts to create these scopes in the IDS environment.